libs/designsystem/src/lib/scss/_global-styles.scss

```diff
@@ -175,6 +175,7 @@ ion-modal.kirby-overlay {
       display: flex;
       justify-content: center;
       align-items: flex-end;
+      pointer-events: none;
```

Comment:
Also do we want the cursor to be `default` when hovering the backdrop instead of `pointer`? It seems like other sites does this - but I suppose it's ultimately up to UX to decide.

Reply:
The modal-wrapper, hierarchy wise, lies infront of the ion-backdrop, meaning we're unable to click the ion-backdrop unless pointer-events of the modal-wrapper is set to none. If there's a more elegent solution I'm all ears 😄  I've played around with the z-index of the modal-wrapper previously, but that turns messy for me really quick.